### ISSUE TITLE
AutoGain is not saved

### ISSUE THREAD

**Issue by agriggio** [artpixls/ART#184]
**[Original report](https://bitbucket.org/agriggio/art/issues/184) by unius (Bitbucket: [unius](https://bitbucket.org/unius), GitHub: [unius](https://github.com/unius)).**

----------------------------------------

Hi!

`I also checked it on the last commit`

**Comment by agriggio** [comment]
**Original comment by Paul Matthijsse (Bitbucket: [Paul Matthijsse](https://bitbucket.org/Paul Matthijsse), ).**

----------------------------------------

Not clear what your problem is. You’re talking about log tone mapping I suppose? Platform?

**Comment by agriggio** [comment]
**Original comment by unius (Bitbucket: [unius](https://bitbucket.org/unius), GitHub: [unius](https://github.com/unius)).**

----------------------------------------

Yes. In “Log Tone Mapping” “Automatic” Gain not saved in .arp. For me, this patch solves the problem

```
--- rtengine/procparams.cc.orig 2021-06-11 01:16:55.000000000 +0300
+++ rtengine/procparams.cc      2021-06-11 01:16:55.000000000 +0300
@@ -3546,7 +3546,7 @@
         if (RELEVANT_(logenc)) {
             saveToKeyfile("LogEncoding", "Enabled", logenc.enabled, keyFile);
             saveToKeyfile("LogEncoding", "Auto", logenc.autocompute, keyFile);
-            saveToKeyfile("LogEncoding", "AutoGray", logenc.autogain, keyFile);
+            saveToKeyfile("LogEncoding", "AutoGain", logenc.autogain, keyFile);
             saveToKeyfile("LogEncoding", "Gain", logenc.gain, keyFile);
             saveToKeyfile("LogEncoding", "TargetGray", logenc.targetGray, keyFile);
             saveToKeyfile("LogEncoding", "BlackEv", logenc.blackEv, keyFile);
```

‌

**Comment by agriggio** [comment]
**Original comment by Paul Matthijsse (Bitbucket: [Paul Matthijsse](https://bitbucket.org/Paul Matthijsse), ).**

----------------------------------------

I see now what you mean. I guess this is due to a recent renaming of some sliders in this tool.

**Comment by agriggio** [comment]
**Original comment by unius (Bitbucket: [unius](https://bitbucket.org/unius), GitHub: [unius](https://github.com/unius)).**

----------------------------------------

Exactly

**Comment by agriggio** [comment]
**Original comment by agriggio (Bitbucket: [agriggio](https://bitbucket.org/agriggio), GitHub: [agriggio](https://github.com/agriggio)).**

----------------------------------------

procparams: properly save logenc.autogain

Fixes #184
